src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.cs

```diff
@@ -198,11 +198,41 @@ public int BytesTransferred
 
         public event EventHandler<SocketAsyncEventArgs>? Completed;
 
+        private void OnCompletedInternal(SocketAsyncEventArgs e)
```

Comment:
This is on a very hot path.  Does it get inlined?  Have you measured to ensure this isn't regressing anything?  It probably isn't, but it's on such an important path it'd be good to double-check.

Reply:
I improved it a bit by removing the argument, as it was always 'this'.

The difference looks like around 1.4 ns per call on my machine, is that a difference we care about on the async path?